XenOvfTransport/Import.cs

```diff
@@ -334,18 +334,20 @@ public void Process(Session xenSession, EnvelopeType ovfObj, string pathToOvf, s
 
                 #region Set vgpu
                 
-                GPU_group gpuGroup;              
-                VGPU_type vgpuType;
-                FindGpuGroupAndVgpuType(xenSession, vhs, out gpuGroup, out vgpuType);
-
-                if (gpuGroup != null)
+                List<Tuple<GPU_group, VGPU_type> >vgpus = FindGpuGroupAndVgpuType(xenSession, vhs);
+                foreach (var item in vgpus)
                 {
-                    var other_config = new Dictionary<string, string>();
+                    GPU_group gpuGroup = item.Item1;
+                    VGPU_type vgpuType = item.Item2;
+                    if (gpuGroup != null)
+                    {
+                        var other_config = new Dictionary<string, string>();
 
-                    if (Helpers.FeatureForbidden(xenSession, Host.RestrictVgpu))
-                        VGPU.create(xenSession, vmRef.opaque_ref, gpuGroup.opaque_ref, "0", other_config);
-                    else if (vgpuType != null)
-                        VGPU.create(xenSession, vmRef.opaque_ref, gpuGroup.opaque_ref, "0", other_config, vgpuType.opaque_ref);
+                        if (Helpers.FeatureForbidden(xenSession, Host.RestrictVgpu))

```

Comment:
If the vGPU feature is restricted, then we create passthrough vGPUs, but we shouldn't create more than one, as the passthrough type doesn't support multiple instances

Reply:
Yes, `passthrough` does not support  multiple at this moment,  the first call will success and the others would just fail. 

It make sense to limit only one vGPU now, and we may need to release this limitation if we can support multiple passthrough. 

I will add a log message stating the vGPU is forbidden and then break the loop tomorrow (I cannot connect to my workstation as the VPN is broken)